rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -293,11 +295,11 @@ def _add_tenant_filter(self, query, model_class, all_tenants):
         if all_tenants:
             # If a user that is allowed to get all the tenants in the system
             # no need to filter
-            if all_tenants_authorization():
+            if all_tenants_authorization(self.current_user):
```

Comment:
Is there actually a current_user attribute on here?

Reply:
Ignore that, I didn't scroll down far enough.

Reply:
there is now!